spacelift/data_stack.go

```diff
@@ -444,6 +444,40 @@ func dataStack() *schema.Resource {
 				Description: "Terraform workspace to select",
 				Computed:    true,
 			},
+			"terragrunt": {
+				Type:        schema.TypeList,
+				Description: "Terragrunt-specific configuration. Presence means this Stack is a Terragrunt Stack.",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"terraform_version": {
+							Type:        schema.TypeString,
+							Description: "The Terraform version.",
+							Computed:    true,
+						},
```

Comment:
Should we set `Optional: true` here? Or Default?

Reply:
I don't think so, because it's the data-source, not the resource. So it'll always be populated.

Reply:
Ah, it's the data source, not a resource. Nevermind.